src/main/charts/confluence/values.yaml

```diff
@@ -286,8 +286,58 @@ volumes:
       # here.
       #
       command:
-  
-  # -- Defines additional volumes that should be applied to all Confluence pods.
+
+  # Each synchrony pod needs its own volume for 'synchrony-home'. Synchrony process will write logs to that location
+  # and any configuration files can be placed there.
+  #
+  synchronyHome:
+
+    # Dynamic provisioning of synchrony-home using the K8s Storage Classes
+    #
+    # https://kubernetes.io/docs/concepts/storage/persistent-volumes/#dynamic
+    # https://atlassian.github.io/data-center-helm-charts/examples/storage/aws/LOCAL_STORAGE/
+    #
+    persistentVolumeClaim:
+
+      # -- If 'true', then a 'PersistentVolume' and 'PersistentVolumeClaim' will be dynamically
+      # created for each pod based on the 'StorageClassName' supplied below.
+      #
+      create: false
+
+      # -- Specify the name of the 'StorageClass' that should be used for the synchrony-home
+      # volume claim.
+      #
+      storageClassName:
+
+      # -- Specifies the standard K8s resource requests and/or limits for the synchrony-home
```

Comment:
I understand that this comment is everywhere in our chart: `Specifies the standard K8s resource requests and/or limits for .. `, but unlike pod/container, pv, pvc and volumeClaimTemplate do not have option for requests.limits. Maybe we can make this comment more tailored for volume claim? At least omit the `and/or limits` part here.

Reply:
Thank you, fixed for all volumes